tests/flow/test_issue_636.py

```
@@ -0,0 +1,32 @@
+from common import *
```

Ruff (0.15.6)

[error] `from common import *` used; unable to detect undefined names

(F403)